elk/lightcurve.py

```diff
@@ -702,12 +721,14 @@ def diagnose_lc_periodogram(self, output_path, freq_bins='auto', identifier='',
             plt.close(fig)
             i += 1
 
-        if save_simbad_queries:
-            np.save(os.path.join(output_path, f'{identifier}_simbad_queries.npy'), simbad_queries)
-
         # convert individual frames to a GIF
         gif_path = os.path.join(output_path, f'{identifier}_pixel_power_gif.gif')
-        with imageio.get_writer(gif_path, mode='I', fps=1.5) as writer:
+        with imageio.get_writer(gif_path, mode='I', fps=2.5) as writer:
             for i in range(len(edges)):
                 writer.append_data(imageio.imread(os.path.join(output_path,
                                                                f'{identifier}_gif_plot_frame_{i}.png')))
+                
+        #get GIF back
+        gif=HTML('<img src="'+gif_path+'">')
```

Comment:
Clever! You could probably use an f-string here too like
```suggestion
        gif=HTML(f'<img src="{gif_path}">')
```